autopts/ptsprojects/zephyr/bap.py

```diff
@@ -148,6 +146,12 @@ def set_addr(addr):
     custom_test_cases = [
         # Errata in progress since the PTS should use
         # TSPX_VS_Company_ID and TSPX_VS_Codec_ID instead.
```

Comment:
This comment seems to belong to `ZTestCase("BAP", "BAP/UCL/SCC/BV-033-C",` and should be moved. 

@sjanc do you know if this is still valid?

Reply:
we don't have any pending erratas for BAP listed, and since that was 2 years ago I'd assume it is fixed by now

Reply:
Ok, I'll remove it

Reply:
I checked my email history,  it was reported against PTS 8.4 and marked as fixed, so likely included in PTS 8.5+

Reply:
Should I then remove all the custom cases that sets the Codec ID, or should that be a separate PR?

Reply:
I'd suggest separate PR